### ISSUE TITLE
Example is for an older version of the SDK

### ISSUE THREAD

**Issue by shanebdavis** [MicrosoftDocs/azure-docs-sdk-node#1002]
[Enter feedback here]

This page's example is out of date. It doesn't show how to use `@azure/arm-storage` with `@azure/identity` or `@azure/ms-rest-nodeauth`. Instead it uses `azure-arm-storage` and `ms-rest-azure`. It starts with:

```javascript
const msRestAzure = require('ms-rest-azure');
const storageManagementClient = require('azure-arm-storage');
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8b7e458e-7cb1-a0ea-380e-9989c2203912
* Version Independent ID: bea79442-2590-5fb2-34b5-77fa9d065e4e
* Content: [Azure Storage modules for JavaScript](https://docs.microsoft.com/en-us/javascript/api/overview/azure/storage-overview?view=azure-node-latest)
* Content Source: [docs-ref-services/latest/storage-overview.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/master/docs-ref-services/latest/storage-overview.md)
* Service: **storage**
* Product: **azure**
* Technology: **azure**
* GitHub Login: @maggiepint
* Microsoft Alias: **magpint**

**Comment by ramya-rao-a** [comment]
Thanks for reporting @shanebdavis, that is a good catch!

Will be fixed with #1009

**Comment by ramya-rao-a** [comment]
The PR #1009 is merged and the content has been updated.
